scripts/prep_data_for_model.R

```diff
@@ -129,12 +134,55 @@ standard_catch_unmarked_trap <- standard_catch_unmarked %>%
 # Summarize releases and recaptures
 standard_recapture %>% glimpse()
 standard_release %>% glimpse()
-gcs_upload(standard_release,
+release_summary <- standard_release |> 
+  mutate(week_released = ifelse(is.na(week_released), week(date_released), week_released),
+         year_released = ifelse(is.na(year_released), year(date_released), year_released)) 
+gcs_upload(release_summary,
            object_function = f,
            type = "csv",
            name = "jpe-model-data/release_summary.csv",
            predefinedAcl = "bucketLevel")
-write_csv(standard_release, "data/model-data/release_summary.csv")
+write_csv(release_summary, "data/model-data/release_summary.csv")
+
+# number of efficiency trials by week
+release_summary_metadata <- release_summary |> 
+  filter(include == "yes") |> 
+  group_by(stream, site, week_released, year_released) |> 
+  distinct(release_id) |> 
+  tally()
+total_trials <- sum(release_summary_metadata$n)
+multiple_trials <- filter(release_summary_metadata, n > 1) 
```

Comment:
Worth letting Josh (and Pete) know in the email that ~42% of weeks have multiple trials. I think Josh thinks that it is more rare than this

Reply:
Not a big deal - especially looking at the flow analysis you did. That is helpful